README.md

```diff
@@ -5,3 +5,8 @@
 - This one is correct
 - This needs a fix
 - This one is correct too
+
+#### markdown-fixes:
```

Comment:
Codacy has a fix for the issue: [Punctuation: ':'](https://app.codacy.com/gh/troubleshoot-codacy/markdown-fixes/pullRequest?prid=9551052)
```suggestion
#### markdown-fixes
```